### ISSUE TITLE
[BUG] _Failure to run lammps minimization by using embedding model in deepmd-kit-2.0.0.beta0-GPU version_

### ISSUE THREAD

**Issue by dfz05** [deepmodeling/deepmd-kit#782]
**I tried to train a DP model with element embedding.  When I used the trained model to run a minimization task in lammps with GPU, the program failed without running. However, if I used the CPU version, the task can run.**

**The version I used is: Deepmd-kit standalone 2.0.0.beta0**

[CrC.zip](https://github.com/deepmodeling/deepmd-kit/files/6693668/CrC.zip)

**~/deepmd-kit/bin/lmp -var compd CrC -in in.minimization**

**The files are in the attachment.**

>

**Comment by amcadmus** [comment]
Could you please paste here the lammps input file? thanks

**Comment by dfz05** [comment]
> Could you please paste here the lammps input file? thanks

The files have been uploaded.

**Comment by njzjz** [comment]
I can't reproduce the bug in my machine (2080 Super) using `v2.0.0.b4`. The output is here
```
     625    109.67568   -605.31012   -605.31012 3.4932874e-08 -2.3933809e-08 -3.3852489e-08    8.1531477    8.1927314    8.1531477
Loop time of 109.676 on 1 procs for 625 steps with 64 atoms

40.6% CPU use with 1 MPI tasks x 1 OpenMP threads

Minimization stats:
  Stopping criterion = force tolerance
  Energy initial, next-to-last, final =
     -605.256198610873  -605.310121712724  -605.310121712724
  Force two-norm initial, final = 6.6132592 5.4833277e-11
  Force max component initial, final = 3.8181670 1.2243286e-11
  Final line search alpha, max atom move = 1.0000000 1.2243286e-11
  Iterations, force evaluations = 625 941

MPI task timing breakdown:
Section |  min time  |  avg time  |  max time  |%varavg| %total
---------------------------------------------------------------
Pair    | 109.63     | 109.63     | 109.63     |   0.0 | 99.95
Neigh   | 0          | 0          | 0          |   0.0 |  0.00
Comm    | 0.0060101  | 0.0060101  | 0.0060101  |   0.0 |  0.01
Output  | 0.014829   | 0.014829   | 0.014829   |   0.0 |  0.01
Modify  | 0          | 0          | 0          |   0.0 |  0.00
Other   |            | 0.02916    |            |       |  0.03

Nlocal:        64.0000 ave          64 max          64 min
Histogram: 1 0 0 0 0 0 0 0 0 0
Nghost:        1267.00 ave        1267 max        1267 min
Histogram: 1 0 0 0 0 0 0 0 0 0
Neighs:        0.00000 ave           0 max           0 min
Histogram: 1 0 0 0 0 0 0 0 0 0

Total # of neighbors = 0
Ave neighs/atom = 0.0000000
Neighbor list builds = 0
Dangerous builds = 0

Please see the log.cite file for references relevant to this simulation

Total wall time: 0:01:51
```

@dfz05 Can you try `v2.0.0.b4`?

**Comment by amcadmus** [comment]
This issue should have been solved in v2.0.0.b4
